loopabull/plugin.py

```diff
@@ -46,4 +46,22 @@ def translate_path(self, routing_key):
         """
         raise NotImplementedError()
 
+    def done(self, result, **kwargs):
+        """
+        a looper plugin can implement this method optionally
+
+        result is a value of loopabull.Result.
+
+        kwargs includes extra variables that depends on the result value.
+        For Result.runerrored, an exitcode kwarg is provided with the ansible
+        exit code.
+        For result.error, an exception kwarg is provided with the exception
```

Comment:
Do you mind making sure to capitalizing `Result.error` and `Exception` in this line for consistency?